### DISCUSSION TITLE
腦科學相關電影推薦！

### DISCUSSION THREAD

**Discussion by Tuoresa** [1101-GS4719-C/Python-Community-GS4719-C#1078] [General]
剛剛介紹了<心中的小星星>
那就順便推薦其他有看過的腦科學相關電影吧
(腦與學習真的是不錯的通識哈哈哈哈)

1. <我想念我自己>
“I am not someone dying. I am someone living with Alzheimer's. I want to
do that as well as I possibly can.”- Still Alice
劇情講述一名語言學專家罹患早發性阿茲海默症的故事，是一部令人蠻衝擊且科學參考價值高的電影，
生命的脆弱與愛的力量都令人痛哭流涕喔買尬
簡介影片：
「請記得我愛你」
https://youtu.be/mPg2BRV66vY

2. <記憶拼圖>
(一部跟我們差不多大的電影)
此部電影的編劇兼導演的克里斯多福·諾蘭，擅長以懸疑、扭曲的方式對於人類的恐懼、罪惡感進行闡述，
他的非線性敘述手法總牽引觀眾思緒與好奇。
因此這部電影雖然講述的是短期記憶失憶症，
卻有一種懸疑片的即視感，看的時候彷彿自己也共同失憶與錯亂XD
對於我來說是非常吸引人的電影節奏！
簡介影片：
https://youtu.be/C593702Y4fY

推薦給大家~
看電影真的很快樂！

**Comment by wenchi9023** [comment]
那我也來推薦! <王牌冤家> 雖然是老片，但真~的超好看!
和原po推薦的類型有些相似，整部以刪除記憶作為電影包裝，然後男女主角穿梭在過往和現實再次找到彼此的故事
滿燒腦的，而且會一直在過去和現在跳來跳去，但很好看! 推薦~~

**Comment by Sylvie0601** [comment]
我也推一個很久以前的日劇!
叫做<ミスター.ブレイン(腦科學先生)>是木村拓哉演的，講男主角在工作的時候遇到了一個嚴重的意外，瀕臨死亡，但在恢復的過程中，腦部急遽發展，讓他擁有超級驚人的思考能力，然後運用腦科學破案的故事!

**Comment by yuxinlin93** [comment]
第二部感覺很好看也
我也來推一部也是諾蘭的電影<全面啟動>，雖然不知道算不算是腦"科學"的，因為她講到有點像意識層面的東西，像是在別人的腦袋裡建築一座意識城市，可以一層一層的進入意識，而且這一部電影的敘事方式非常有趣，看完非常佩服編劇，非常值得一看
